tests/test_base.py

```diff
@@ -37,7 +37,7 @@ def __exit__(self, *args, **kwargs):
 
 
 class ConfigGetterTestCase(unittest.TestCase):
-    example_directory = os.path.join(os.path.dirname(__file__), 'config/')
+    example_directory = os.path.abspath(os.path.join(os.path.dirname(__file__), 'config/'))
```

Comment:
This is an unrelated change. Many unit tests failed in my virtual environment because `os.path.dirname(__file__)` resolved to the relative path `tests/` and not to `/home/carl/code/getconf/tests/`. Using `abspath` fixes the problem.